tests/dummy/app/controllers/application.js

```diff
@@ -7,7 +7,7 @@ export default class ApplicationController extends Controller {
   constructor(...args) {
     super(...args);
     let canton = localStorage.getItem("canton");
-    if (!canton) {
+    if (!canton || canton === "null") {
       //eslint-disable-next-line no-alert
       canton = prompt("canton (BE|SZ)");
       localStorage.setItem("canton", canton);
```

Comment:
localStorage.setItem seems to store null values as "null". Wouldn't it make sense to not set the item in the localStorage if it isn't within the expected values (BE, SZ, UR, GR, SO)? The next time we try to read the localStorage canton it will actually be null.

Reply:
Added a null check before `localStorage.setItem(...)`. This way we won't produce new instances of this error but also catch the case someone still has `"null"` in local storage. ok?

Reply:
Sounds good to me :+1: